cmd/server-main.go

```diff
@@ -310,6 +310,14 @@ func serverMain(ctx *cli.Context) {
 	globalObjectAPI = newObject
 	globalObjLayerMutex.Unlock()
 
+	err = globalObjectAPI.FixDeploymentID(context.Background())
+
+	if err != nil && (err != NotImplemented{}) {
```

Comment:
`err` will never be equal to `NotImplemented{}`, `NotImplemented{}` is a new object in memory